app/styles/utils.scss

```diff
@@ -80,18 +94,49 @@ $units: 0, 1, 2, 3, 4;
 
 .font-bold { font-weight: bold; }
 .font-mono { font-family: var(--font-monospace); }
+.leading-none { line-height: 1; }
 .no-underline { text-decoration: none; }
+.underline { text-decoration: underline; }
 .text-left { text-align: left; }
 .text-center { text-align: center; }
 .text-right { text-align: right; }
 .whitespace-no-wrap { white-space: nowrap; }
 
 .bg-transparent { background-color: var(--transparent); }
-.bg-base00 { background-color: var(--base00); }
-.bg-base01 { background-color: var(--base01); }
-.bg-base02 { background-color: var(--base02); }
+
+.fill-current { fill: currentColor; }
 
 .cursor-default { cursor: default; }
 .cursor-pointer { cursor: pointer; }
+.cursor-col-resize { cursor: col-resize; }
 
 .z-10 { z-index: 10; }
+
+.appearance-none {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+.pointer-events-none { pointer-events: none; }
+
+.outline-none { outline: 0; }
+
+/*
+  Utilities not part of Tailwind:
+*/
+
+.text-inherit { color: var(--inherit); }
+
+.bg-base00 { background-color: var(--base00); }
+.bg-base01 { background-color: var(--base01); }
+.bg-base02 { background-color: var(--base02); }
+.bg-base05 { background-color: var(--base05); }
+.bg-spec00 { background-color: var(--spec00); }
+
+.text-base11 { color: var(--base11); }
+.text-base13 { color: var(--base13); }
+
+.text-12 { font-size: 12px; }
```

Comment:
I'm not sure how many text sizes we have, but maybe we should try to standardize on `text-xs` `text-sm` etc to align with tailwind?

Reply:
I've been thinking about this. Since we have an app the spectrum of font sizes goes something like: `font-size: 10px – 15px`. Not sure it makes sense to have names like this:

```
```

Reply:
I think your suggestion sounds fine:

```css
text-xs: 10px
text-sm: 11px
text-base: 12px
text-lg: 13px
text-xl: 14px
```

It is a little bit odd to have them all 1px different though. I think I would advocate more for setting a base font size and making all these `rem` based like tailwind does.